main.go

```diff
@@ -33,6 +33,7 @@ var (
 	name                 string
 	metricsPort          string
 	alertSeverity        string
+	alertEnabled         bool
```

Comment:
Why does this need to be a var? It's part of the *cli.Context and should just pulled from there